sipn_reanalysis_ingest/luigitasks/convert.py

```diff
@@ -38,67 +44,112 @@ class Grib2ToNcDaily(luigi.Task):
 
     date = luigi.DateParameter()
 
+    @property
+    def uses_5day_tars(self):
+        if self.date < CFSR_DAILY_TAR_BEFORE:
+            return True
+        return False
```

Comment:
This function is useful, but maybe we should reconsider it. It only has two outcomes: True or False. But there are 3 cases: We either need only 5-day tars, or we need only 1-day tars, or for the case where `self.date == CFSR_DAILY_TAR_BEFORE`, we need both.

Maybe this function should return a dictionary like this that specifies all possible outcomes (both `False` should be impossible):

```
return { '5-day': True, '1-day': True}
```

Or an Enum?:

```
class DailyTarsRequired(Enum):
    FIVE_DAY = 'five_day'
    DAILY = 'daily'
    BOTH = 'both'
```

```
return DailyTarsRequired.BOTH
```

Reply:
Done. In _types.py.